src/Links/Sessions.ts

```diff
@@ -64,10 +64,10 @@ export class Links extends Resource {
   }
 
   /**
-   * Generate a Duffel Link per the configuration
+   * Create a Duffel Links Session per the configuration
    */
-  public generateLink = async (
-    data: GenerateLinkParameters
-  ): Promise<DuffelResponse<GeneratedLink>> =>
-    this.request({ method: 'POST', path: `${this.path}/generate_link`, data })
+  public create = async (
```

Comment:
nit: I wonder if this should be call `createSession` for clarity 🤔

Reply:
Move this to a `Sessions` class, it's now `Sessions.create`